### DISCUSSION TITLE
Welcome to iaqualink-py Discussions!

### DISCUSSION THREAD

**Discussion by flz** [flz/iaqualink-py#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rkinnc75** [comment]
Hi,
  I am an absolute newbie to python development, but have picked up a bit here and there.

  I have a Jandy pump that I'd like to automate more with my solar panels and the info from tomorrow.io API.  I found this project and would love to find out if I can turn off the pump, turn on the pump, etc. based on the data I get from the API as to whether or not it's sunny or cloudy.  If it's sunny, I can run the pump all day and no electric bill, etc.

**Reply by rkinnc75** [reply]
When I created my test script from the README.md, I get an error.  I did replace the email/password in my script, but have put it back for the discussion example.

pi@raspberrypi:~ $ python
Python 3.9.2 (default, Mar 12 2021, 04:06:34) 
[GCC 10.2.1 20210110] on linux

**Reply by rkinnc75** [reply]
OK, I've made some minor changes, but get a different error:  i2d is not a supported system type.

pi@raspberrypi:~ $ cat aqua.py 
```
#!/usr/bin/python

import iaqualink
from iaqualink.client import AqualinkClient
import asyncio

async def main():
    async with AqualinkClient('xxx@example.com', 'password') as c:
        print(c)
        s = await c.get_systems()
        print(s)
        d = await list(s.values())[0].get_devices()
        print(d)

if __name__=="__main__":
    asyncio.run(main())
```


pi@raspberrypi:~ $ python aqua.py 
```
<iaqualink.client.AqualinkClient object at 0xb5cfb370>
i2d is not a supported system type.
{}
Traceback (most recent call last):
  File "/home/pi/aqua.py", line 16, in <module>
    asyncio.run(main())
  File "/usr/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/pi/aqua.py", line 12, in main
    d = await list(s.values())[0].get_devices()
IndexError: list index out of range
```
